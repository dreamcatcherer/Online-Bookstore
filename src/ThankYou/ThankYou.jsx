import React from "react";
import thankYou from "../images/ThankYou.jpg";
import styles from "./ThankYou.module.css";

const ThankYou = () => {
  return (
    <div className={styles.container}>
      <img src={thankYou} alt="Thank you" className={styles.image} />
      
     <div className={styles.text}>
     <b> Thank you for visiting our bookstore! </b>
    </div>
    </div>
  );
};

export default ThankYou;
