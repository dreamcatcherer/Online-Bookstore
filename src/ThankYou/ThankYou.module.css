.container {
  display: grid;
  grid-template-columns: 1fr;
}

.image {
  align-self: center;
  justify-self: center;
}

.text{
	text-align: center;
	justify-content: center;
}
