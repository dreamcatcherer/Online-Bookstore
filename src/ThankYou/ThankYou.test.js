import React from "react";
import { render } from "@testing-library/react";
import ThankYou from "./ThankYou.jsx";

it("renders the ThankYou component correctly", () => {
  const { container } = render(<ThankYou />);
  expect(container).toMatchSnapshot();
});
