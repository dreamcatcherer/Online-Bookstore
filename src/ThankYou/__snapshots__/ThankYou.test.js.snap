// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the ThankYou component correctly 1`] = `
<div>
  <div
    class="container"
  >
    <img
      alt="Thank you"
      class="image"
      src="ThankYou.jpg"
    />
    <div
      class="text"
    >
      <b>
         Thank you for visiting our bookstore! 
      </b>
    </div>
  </div>
</div>
`;
